tests/cupy_tests/math_tests/test_sumprod.py

```diff
@@ -54,7 +59,7 @@ def test_sum_axis(self, xp, dtype):
     def test_sum_axis_huge(self, xp):
         a = testing.shaped_random((2048, 1, 1024), xp, 'b')
         a = xp.broadcast_to(a, (2048, 1024, 1024))
-        return a.sum(axis=0)
+        return a.sum(axis=2)
```

Comment:
For backport, only this fix is needed.
There're not default memory pools in CuPy v2.

Reply:
Thank you for pointing out. Fixed.